Directory.Build.props

```diff
@@ -29,7 +29,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <CopyDocumentationFilesFromPackages>true</CopyDocumentationFilesFromPackages>
     <CopyDebugSymbolFilesFromPackages>true</CopyDebugSymbolFilesFromPackages>
-    <NoWarn>$(NoWarn);1591;S3267</NoWarn>
+    <NoWarn>$(NoWarn);1591;S3267;S1133;S4144</NoWarn>
```

Comment:
I really don't think that simply ignoring the warnings is the best way to resolve them. What about getting rid of deprecated code and identical implementations instead?